### ISSUE TITLE
ssm agent pre installed for aws images

### ISSUE THREAD

**Issue by kevmando** [gardenlinux/gardenlinux#187]
**What would you like to be added**:
aws ssm agent 

**Why is this needed**:
similar as #179, My company requires tenable.io vulnerability scan with AWS frictionless scan method, which requires ssm agent.
Also it may help to manage EC2 instances with SSM features as well in other use cases.

https://docs.tenable.com/tenableio/vulnerabilitymanagement/Content/Settings/Connectors/AWS/FrictionlessAssessment.htm
https://github.com/aws/amazon-ssm-agent

**Comment by vrenjith** [comment]
In order to enable SSH access into the nodes in AWS Gardener cluster, this agent is needed.

**Comment by marwinski** [comment]
No, the agent is not needed to get ssh access. Please review https://github.com/gardener/gardenctl or https://github.com/gardener/gardenctl-v2 and get in contact with the authors.

**Comment by vrenjith** [comment]
To get access that respects AWS IAM, don't we need this Agent?
